zingo-testutils/src/chain_generic_tests.rs

```diff
@@ -8,11 +8,63 @@ use zingolib::wallet::notes::query::OutputQuery;
 use zingolib::wallet::notes::query::OutputSpendStatusQuery;
 use zingolib::{get_base_address, wallet::notes::query::OutputPoolQuery};
 
+/// runs a send-to-receiver and receives it in a chain-generic context
+pub async fn propose_and_broadcast_value_to_pool<TE>(send_value: u32, pooltype: PoolType)
+where
+    TE: ConductChain,
+{
+    let mut environment = TE::setup().await;
+
+    dbg!("chain set up, funding client now");
```

Comment:
I feel quite stongly about dbg! NEVER making it into dev. I use `println!` in test where i think there should be relavent information in the console when a test fails or runs with --no-capture

Reply:
or even using logs and assigning it to debug level is a good compromise?

Reply:
> or even using logs and assigning it to debug level is a good compromise?

Logs aren't viewable from tests, unfortunately. Agree that we shouldn't have dbg!s in non-test code.